typescript/sdk/src/consts/chainMetadata.ts

```diff
@@ -691,6 +691,46 @@ export const gnosis: ChainMetadata = {
   gnosisSafeTransactionServiceUrl: 'https://safe-transaction.xdai.gnosis.io/',
 };
 
+export const solanadevnet: ChainMetadata = {
+  chainId: 13375,
+  name: Chains.solanadevnet,
+  protocol: ProtocolType.Sealevel,
+  displayName: 'Solana Devnet',
+  nativeToken: {
+    decimals: 9,
+    name: 'Solana',
+    symbol: 'SOL',
+  },
+  publicRpcUrls: [{ http: 'https://api.devnet.solana.com' }],
+  blockExplorers: [],
+  blocks: {
+    confirmations: 0,
```

Comment:
Just confirming these shouldn't be 1?

Reply:
i didn't really put thought into it here, I'll change

Fwiw the Rust SDK/client lets you specify a commitment config / commitment level https://docs.rs/solana-sdk/latest/solana_sdk/commitment_config/struct.CommitmentConfig.html so the confirmations are almost an EVM smell. Not sure if there's something similar in TS

Reply:
Cool thanks.
Just to clarify, the confirmations field is what things like the timeline widget use to calculate estimates for the delivery stages of a message